src/gateway/how-to/kong-gateway.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Run Kong Gateway in minutes"
+description: "A how-to guide for quickly starting a Kong Gateway"
+---
+
+In order to explore the capabilities of [{{site.base_gateway}}](/gateway), 
+you'll need one to experiment with. This guide helps you quickly deploy Kong 
+using [Docker](https://docs.docker.com/get-started/overview/) which is the 
+easiest way to get started. This guide's purpose is not to provide a production like deployment
+or explanation of deployment steps, rather to quickly get you a running gateway and an 
+example [service](/gateway/admin-api/#service-object) as quickly as possible.
+
+### Prerequisites
+
+This guide assumes each of the following tools are installed locally. 
+* [Docker](https://docs.docker.com/get-docker/) is used to run Kong and the supporting database locally. This guide has been tested with version `20.10.17`.
+* [curl](https://curl.se/) is used to send requests to the gateway. Most systems come with `curl` pre-installed.
+
+### Steps 
+
+In order to get started quickly, you'll download and run a bash script which contains 
+commands to run Kong, it's supporting database, and an example service to work with.
+Then you'll interact with the gateway using `curl` to ensure it has been started properly.
+
+Run the following command to start {{site.base_gateway}} using Docker:
+
+```sh
+curl -Ls get.konghq.com/quickstart | sh -s
+```
+
+{:.note}
+> **Note:** The script creates a log file in the current directory named `kong-quickstart.log`
+
+Docker is now downloading and running the {{site.base_gateway}} and supporting database. Additionally,
+the script bootstraps the database and installs a [mock service](https://mockbin.org/) to experiment with.
+Depending on your internet download speeds, this command should complete relatively quickly, and once you have the images cached locally, subsequent usage of this guide will complete much faster.
+
+Once Kong is available, you will see:
```

Comment:
This change concerns me that we don't clarify to the user that the port numbers displayed are an example, otherwise they may get confused with the difference in their output or think that the port numbers we document are the ones they should use.  Below I use the phrase "should look similar to the following" which IMO underscores the ports may be different for them.